libraries/core-react/src/Banner/BannerMessage.tsx

```diff
@@ -1,12 +1,16 @@
-// @ts-nocheck
 import React from 'react'
-import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { Typography } from '../Typography'
+import { Typography  } from '../Typography'
+import { Props as TypographyProps } from '../Typography/Typography'
 
 const StyledBannerMessage = styled(Typography)``
 
-export const BannerMessage = ({ children, ...props }) => {
+type Props = {
+  /** Text content */
+  children: string
+} & Omit<TypographyProps,'children'>
```

Comment:
Note here the forcing of children to be a string, but also enabling easy props view for the props that will be sent further to typography.